src/configSchema.ts

```diff
@@ -433,6 +445,34 @@ export const VALIDATION_SCHEMA = z
 			),
 		ZodErrorMessages.injectUrl,
 	)
+	.refine((config) => {
+		if (
+			config.delugeRpcUrl &&
+			config.blockList.some((b) => b.startsWith(`${BlocklistType.TAG}:`))
+		) {
+			logger.error(
+				`${BlocklistType.TAG}: blocklisting is deprecated for Deluge, please use ${BlocklistType.CATEGORY}: instead (all ${BlocklistType.TAG}: blocklist items has being automatically converted to ${BlocklistType.CATEGORY}:).`,
+			);
+			config.blockList = config.blockList.map((b) => {
+				if (b.startsWith(`${BlocklistType.TAG}:`)) {
+					return b.replace(BlocklistType.TAG, BlocklistType.CATEGORY);
+				}
+				return b;
+			});
+		}
+		if (
```

Comment:
I wouldn't worry about backwards compatibility as much in this situation because it's very new. there's an argument it's just a bug fix

Reply:
Fine either way though

Reply:
This is no cost to do since it's handling during validation and can be removed in a few versions.